docs/src/api/class-locator.md

```diff
@@ -871,6 +871,37 @@ If [`param: expression`] throws or rejects, this method throws.
 
 **Usage**
 
+Passing argument to [`param: expression`]:
+
+```js
+const result = await page.getByTestId('myId').evaluate((element, [x, y]) => {
```

Comment:
It's unclear why one would do locator.evaluate rather than page.evaluate for this. Add a code that uses the `element`?